utilities/llmd_utils.py

```
@@ -0,0 +1,658 @@
+    model_name: str,
+) -> None:
+    """Validate response for authorized users."""
+
+    use_regex = False
+
+    if use_default_query:
+        expected_response_text_config = inference_config.get("default_query_model", {})
+        use_regex = expected_response_text_config.get("use_regex", False)
+
+        if not expected_response_text_config:
+            raise ValueError(f"Missing default_query_model config for inference {inference_config}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)